crates/sui-config/src/genesis.rs

```diff
@@ -286,6 +298,14 @@ pub struct GenesisChainParameters {
     /// protocol version that the chain starts at.
     #[serde(default = "ProtocolVersion::max")]
     pub protocol_version: ProtocolVersion,
+
+    #[serde(default = "GenesisChainParameters::default_allow_insertion_of_extra_objects")]
+    pub allow_insertion_of_extra_objects: bool,
+
+    /// The initial account address that will own the initial 9 Billion Sui that is minted at
+    /// genesis.
+    #[serde(default)]
+    pub initial_account_address: SuiAddress,
```

Comment:
Maybe make this name more descriptive like `initial_sui_custody_account_address`?

Reply:
naming is so hard, thanks for the suggestion on a better name